modules/imgproc/src/resize.cpp

```diff
@@ -1796,8 +1800,73 @@ struct HResizeLinearVecU8_X4
     }
 };
 
+#ifdef __ARM_FEATURE_SVE
+// Accuracy test at: ./bin/opencv_test_imgproc --gtest_filter="Imgproc_Resize*"
+struct HResizeLinearVec_16u32f_SVE
+{
```

Comment:
Original code is a template for all types.
This variant is targeted for single types set.

OpenCV SIMD UI is designed to support templates.